service/lib/agama/storage/proposal_settings_conversion.rb

```diff
@@ -45,6 +47,24 @@ def self.from_y2storage(y2storage_settings, settings)
       def self.to_y2storage(settings, config:)
         ToY2Storage.new(settings, config: config).convert
       end
+
+      # Performs conversion from Hash according to the JSON schema.
+      #
+      # @param schema_settings [Hash]
```

Comment:
What "schema" means? For me, the "schema" is the JSON specification. On IRC we decided to address this issue later.